ledger-core/v2/virtual/integration/constructor_test.go

```diff
@@ -194,6 +195,10 @@ func TestVirtual_Constructor_WithExecutor(t *testing.T) {
 
 		server.SendMessage(ctx, msg)
 
-		<-testIsDone
+		select {
+		case <-time.After(3 * time.Second):
```

Comment:
3 seconds is often way to low for CI environments

Reply:
It is set on other tests to 3 seconds and was not triggering fails for now. I'm not sure whether it is better to have it in single constant or could be set individually for each test. If tests will start flapping we will adjust timeouts

Reply:
updated timeout to be 10 seconds.